codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2WindowUpdateFrame.java

```diff
@@ -32,8 +32,8 @@ public DefaultHttp2WindowUpdateFrame(int windowUpdateIncrement) {
     }
```

Comment:
So you say we should remove the check here and just let `Http2ConnectionHandler` handle it ?